.betterer.results

```diff
@@ -7798,11 +7798,36 @@ exports[`better eslint`] = {
       [0, 0, 0, "Unexpected any. Specify a different type.", "0"],
       [0, 0, 0, "Unexpected any. Specify a different type.", "1"]
     ],
+    "public/app/plugins/datasource/mssql/query_ctrl.ts:5381": [
+      [0, 0, 0, "Unexpected any. Specify a different type.", "0"],
+      [0, 0, 0, "Unexpected any. Specify a different type.", "1"],
+      [0, 0, 0, "Unexpected any. Specify a different type.", "2"],
+      [0, 0, 0, "Unexpected any. Specify a different type.", "3"]
+    ],
```

Comment:
hmmm this doesn't seem right - as recent PR removed these I think. Looks like you merged main in just now so little bit confused about this?

`yarn betterer` [`-u`?] should fix this 😌

Reply:
Maybe I did something wrong, but I:
1. Created this PR
2. Betterer failed https://drone.grafana.net/grafana/grafana/70784 on files I haven't touched
3. I ran `yarn betterer -u` to fix failing betterer
4. I merged origin/main
5. I ran `yarn betterer -u` to see if these changed will be removed, but nothing happened

I also tried to re-run `yarn betterer`, `yarn betterer -u` as you mentioned, just now, but there are no changes to `betterer.results` file. 
<img width="687" alt="image" src="https://user-images.githubusercontent.com/30407135/178693458-182e9106-5ab8-448f-998d-d073db6e496a.png">

Reply:
hmmm actually you're right. they should be there by the state of main. no idea how they've gotten out of sync :( 

I guess this is the drawback of allowing betterer CI step to fail :(

I think your PR is correct, these changes should be comitted.